### ISSUE TITLE
add AttVamp to C sheet

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#345]
We got plenty of space at third C pannel - right now there is only ESP in 1st column

/rfe add second column at third C sheet where to include AttVamp.

Atm I play necro and it's hard to understand does my form got vamp or not (eg Vampire Bat), while I see AttVamp status in buff bottom main UI when I case Shadow Touch spell

**Comment by draconisPW** [comment]
You're right, and in this case why not add a column for slays and a column for brands. This will be easier to see if not only player gets a specific brand (+), a temporary brand (!), or even an off-weapon brand that is eaisly overlooked.
